tools/generator/test/CreateAutogeneratedXCSchemesTests.swift

```diff
@@ -172,6 +171,13 @@ class CreateAutogeneratedXCSchemesTests: XCTestCase {
                 hostIndex: hostIndex
             ),
         ]
+        if expectedTargetInfo.productType.isLaunchable {
```

Comment:
We should only do this for `buildMode != .xcode`, right? There shouldn't be a need for Xcode built apps.

Reply:
Good point! Yeah, since we are not using a custom clang for BwX mode, we don't need it. But if there is a CC override in BwX mode, we will need a similar setup there as well. 

I updated the PR to only add this support for BwB mode since all this setup needs to have the same relative structure that Xcode expects.